code/net/ConnectX-5/mlx5.c

```diff
@@ -676,7 +726,7 @@ bool tn_net_device_init(const struct tn_pci_address pci_address, struct tn_net_d
   // OPCODE_SET_ISSI - Table 1267
   ((volatile uint32_t *) command_queues_virt_addr)[4] = le_to_be_32(0x010B0000);
   // OPCODE_SET_ISSI - Table 1267
-  ((volatile uint32_t *) command_queues_virt_addr)[6] = current_issi;
+  ((volatile uint32_t *) command_queues_virt_addr)[6] = le_to_be_32(1); // or minimum_issi (a.k.a 601)?
```

Comment:
For some reason it works to set the ISSI value to 1, but when I try to set the ISSI to the minimum_issi computed above, SET_ISSI return status is 0x03, _Parameter not supported, parameter out of range, reserved not equal 0_

Reply:
Should I set the ISSI to 1 ?